docs/03-tutorials/00-application-connectivity/ac-02-get-client-certificate.md

```diff
@@ -83,18 +87,28 @@ A successful call returns the following response:
 }
 ```
 
-> **NOTE:** The response contains URLs to the Application Registry API and the Events Publisher API, however, it is not recommended to use them. You should call the `metadata` endpoint URL, which is provided in `api.infoUrl` property, to fetch correct URLs to the Application Registry API and to the Events Publisher API, and other configuration details.
+> **NOTE:** The response contains URLs to the Application Registry API and the Events Publisher API, however, using them is not recommended. To fetch correct URLs to these APIs, as well as other configuration details, call the `metadata` endpoint URL. It is provided in the **api.infoUrl** property.
 
 ## Generate a CSR and send it to Kyma
 
-1. Generate a CSR using the certificate subject data obtained in the previous step:
+1. Export the names of the generated CSR, client certificate and key, and your [cluster domain](../../02-get-started/01-quick-install.md#export-your-cluster-domain) as environment variables:
+
+   ```bash
+   export CSR_FILE_NAME=generated
+   export CLIENT_CERT_FILE_NAME=generated
+   export KEY_FILE_NAME=generated
+   export CLUSTER_DOMAIN=local.kyma.dev
+   ```
+   
+2. Generate a CSR using the certificate subject data obtained in the previous step:
 
    ```bash
    openssl genrsa -out generated.key 2048
-   openssl req -new -sha256 -out generated.csr -key generated.key -subj "/OU=OrgUnit/O=Organization/L=Waldorf/ST=Waldorf/C=DE/CN={APP_NAME}"
-   openssl base64 -in generated.csr
+   openssl req -new -sha256 -out $CSR_FILE_NAME.csr -key $KEY_FILE_NAME.key -subj "/OU=OrgUnit/O=Organization/L=Waldorf/ST=Waldorf/C=DE/CN=$APP_NAME"
+   openssl base64 -in $CSR_FILE_NAME.csr -A
```

Comment:
I added `-A` here because it gets rid of the newlines in the printed CSR, which I otherwise have to remove manually.